sdk/ml/azure-ai-ml/azure/ai/ml/entities/_credentials.py

```diff
@@ -692,9 +745,9 @@ def __ne__(self, other: object) -> bool:
 class AccessKeyConfiguration(RestTranslatableMixin, DictMixin):
     """Access Key Credentials.
 
-    :param access_key_id: access key id
+    :param access_key_id: The access key ID.
     :type access_key_id: str
-    :param secret_access_key: secret access key
+    :param secret_access_key: The secrete access key.
```

Comment:
```suggestion
    :param secret_access_key: The secret access key.
```

Reply:
Good catch! The edge case of a spell-checker. Sometimes the word is real, but still not right...